apps/angularlogos/src/app/app.component.html

```diff
@@ -28,11 +35,14 @@ <h1><a routerLink="/">Angular Logos</a></h1>
     <div class="credits">
       <hr />
       <p>
-        Angular Logos - Created by <a href="https://mobile.twitter.com/maartentibau" target="_blank">Maarten Tibau</a> &amp;
-        <a href="https://twitter.com/fmalcher01" target="_blank">Ferdinand Malcher</a><br />
+        <strong>Angular Logos</strong> – Created with ❤️ by
+        <a href="https://twitter.com/maartentibau" target="_blank" rel="noreferrer">Maarten Tibau</a> &amp;
+        <a href="https://twitter.com/fmalcher01" target="_blank" rel="noreferrer">Ferdinand Malcher</a><br />
         <br />
-        <a href="https://angularlogos.com">GitHub (Gallery)</a> -
-        <a href="https://github.com/maartentibau/angular-logos" target="_blank">GitHub (Logos)</a>
+        <mat-icon fontSet="fab" fontIcon="fa-github"></mat-icon>
```

Comment:
`mat-icon` is a bit overkill here I think, I would go with
`<i class="fab fa-github"></i> <strong>Github: <strong>`

Reply:
Ah good to know, what's the difference?

Reply:
Well, `<i>` is just a native HTML tag for icons, `mat-icon` is the Angular Material icon component, which I actually only use within AM components them self, like `mat-button`.

But I think just using an `<i>` tag in there as well might also work, haven’t tried that one yet.

Reply:
Okay! I know both ways but always thought the `mat-icon` is just a handy wrapper. Will change this to `<i>`!